features/step_definitions/networking.rb

```diff
@@ -1122,4 +1122,40 @@
     step %Q/the outputs should contain "sctp"/
     $i +=1
   end
-end
\ No newline at end of file
+end
+
+Given /^get egress data file directory to the#{OPT_SYM} clipboard$/ do | cb_name |
```

Comment:
this step's wording doesn't sound correct to me.  Perhap add `I` or `admin` to the beginning

Reply:
Fixed it, thanks!